eng/pipelines/templates/VS-release.template.yml

```diff
@@ -23,6 +29,20 @@ steps:
 
 - template: ../steps/CopyAndPublishSymbols.yml
 
+- script: |
+    mkdir $(Build.StagingDirectory)\zips\
+  displayName: "Create $(Build.StagingDirectory)/zips/ folder"
+
+- template: ../steps/PublishOpenDebugAD7.yml
+  parameters:
+    Configuration: 'Lab.Release'
+    RuntimeID: 'win-x64'
+    SignedBinariesFolder: '$(Build.StagingDirectory)\drop'
+
+- powershell: | 
+    Copy-Item -Recurse '$(Build.StagingDirectory)\debugAdapters\win-x64\*' -Destination '$(Build.StagingDirectory)\drop\Release\vscode\.'
+  displayName: "Copy OpenDebugAD7 to drop"
+
```

Comment:
As discussed, I think we probably don't want to do this for VS, or at least we don't want to publish it as self-contained.